packages/react/src/index.ts

```diff
@@ -245,5 +245,11 @@ export * from './lib/accessibility/FocusZone/FocusTrapZone.types'
 export * from './lib/accessibility/FocusZone/AutoFocusZone.types'
 export * from './lib/accessibility/types'
 export * from './lib/accessibility/reactTypes'
+// Dev-time accessibility attributes integrity validation.
+import { setup } from '@stardust-ui/ability-attributes'
+
+if (process.env.NODE_ENV !== 'production') {
+  setup()
+}
```

Comment:
Can we do this only for doc site, now?
Do we really want to ship this stuff to consumers as default now?